packages/docusaurus-init/templates/classic/sidebars.js

```diff
@@ -1,6 +1,16 @@
 module.exports = {
-  someSidebar: {
-    Docusaurus: ['doc1', 'doc2', 'doc3'],
-    Features: ['mdx'],
-  },
+  docs: [
+    {
+      type: 'category',
+      label: 'Docusaurus Tutorial',
+      items: [
+        'getting-started',
+        'markdown-features',
```

Comment:
this md feature page is probably better at the end? I feel the user should first create pages before having more details about MD